CHANGES.rst

```diff
@@ -1,3 +1,95 @@
+5.0.1 (2022-01-25)
+==================
+
+Bug Fixes
+---------
+
+astropy.coordinates
+^^^^^^^^^^^^^^^^^^^
+
+- Trying to create an instance of ``astropy.coordinates.Distance`` by providing
+  both ``z`` and ``parallax`` now raises the expected ``ValueError``. [#12531]
+
+- Fixed a bug where changing the wrap angle of the longitude component of a
+  representation could raise a warning or error in certain situations. [#12556]
+
+- ``astropy.coordinates.Distance`` constructor no longer ignores the ``unit``
+  keyword when ``parallax`` is provided. [#12569]
+
+astropy.cosmology
+^^^^^^^^^^^^^^^^^
+
+- ``astropy.cosmology.utils.aszarr`` can now convert ``Column`` objects. [#12525]
+
+- Reading a cosmology from an ECSV will load redshift and Hubble parameter units
+  from the cosmology units module. [#12636]
+
+astropy.io.fits
+^^^^^^^^^^^^^^^
+
+- Fix formatting issue in ``_dump_coldefs`` and add tests for ``tabledump`` and
+  ``tableload`` convenience functions. [#12526]
+
+astropy.io.misc
+^^^^^^^^^^^^^^^
+
+- YAML can now also represent quantities and arrays with structured dtype,
+  as well as structured scalars based on ``np.void``. [#12509]
+
+astropy.modeling
+^^^^^^^^^^^^^^^^
+
+- Fixes error when fitting multiplication or division based compound models
+  where the sub-models have different output units. [#12475]
+
+- Bugfix for incorrectly initialized and filled ``parameters`` data for ``Spline1D`` model. [#12523]
+
+- Bugfix for ``keyerror`` thrown by ``Model.input_units_equivalencies`` when
+  used on ``fix_inputs`` models which have no set unit equivalencies. [#12597]
+
+astropy.table
+^^^^^^^^^^^^^
+
+- ``astropy.table.Table.keep_columns()`` and
+  ``astropy.table.Table.remove_columns()`` now work with generators of column
+  names. [#12529]
```

Comment:
Might be formatted nicer (remove second line break of push it before "now work"), but that probably should have been done in the original entry.